src/lib/Quiz.jsx

```diff
@@ -1,8 +1,9 @@
+/*eslint-disable*/
 import React, { useState, useEffect, useCallback } from 'react';
 import Core from './Core';
 import defaultLocale from './Locale';
 import './styles.css';
-
+import quiz from '../docs/quiz';
```

Comment:
this is not correct. docs is the app using `lib`. you are making infinite reference here. please remove this line.